services/provider/server/server.go

```diff
@@ -193,7 +193,17 @@ func (s *OCSProviderServer) GetStorageConfig(ctx context.Context, req *pb.Storag
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "Failed to construct status response: %v", err)
 		}
-		desiredClientConfigHash := getDesiredClientConfigHash(channelName, consumerObj)
+
+		storageCluster, err := s.getStorageCluster(ctx)
+		if err != nil {
+			return nil, err
```

Comment:
The error the getStorageCluster is returning is already formated in gRPC conventions so we can directly return it here instead of reformating it again.

Reply:
ack, resolve comment.